### PR TITLE
added support for update_password=on_create for ansible_version 2.0+

### PR DESCRIPTION
**Pull request by solomongifford**

### CODE REVIEW

**Comment by geerlingguy** [comment]
@solomongifford - Is this going to be available in 1.9.x? I notice it was merged into the Ansible `devel` branch, and has `version_added: "1.9"` in the PR...

**Comment by solomongifford** [comment]
There's a commit right after that changing the release to 2.0.  The merge request was made before 1.9 but accepted after 1.9.

File: defaults/main.yml [hunk 1]
```diff
@@ -66,7 +66,8 @@ mysql_databases: []
 mysql_users: []
 # Full Example:
 # mysql_users:
-#   - { name: example, host: 127.0.0.1, password: secret, priv: *.*:USAGE }
+#   - { name: example, host: 127.0.0.1, password: secret, update_password: allways,  priv: "*.*:USAGE" }
```
Comment (JorisVanEijden) [inline defaults/main.yml]: 'allways' should be 'always'

File: tasks/users.yml [hunk 2]
```diff
@@ -7,3 +7,15 @@
...
+  when: ansible_version.full | version_compare('2.0', '<')
+
+- name: Ensure MySQL users are present.
+  mysql_user:
+    name: "{{ item.name }}"
+    host: "{{ item.host | default('localhost') }}"
+    password: "{{ item.password }}"
+    priv: "{{ item.priv | default('*.*:USAGE') }}"
+    state: present
+    update_password: "{{ item.update_password }}"
```
Comment (JorisVanEijden) [inline tasks/users.yml]: No default provided.

**Comment by solomongifford** [comment]
This is likely no longer an issue due to the fact the module no longer executes the mysql_user after the first run.

**Pull request closed without merging**